cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -136,6 +238,29 @@ def derive_work_status_columns(df: DataFrame) -> DataFrame:
     # ["contact_participant_hospital", "contact_other_in_hh_hospital"])
     # df = placeholder_for_derivation_number_10(df, "contact_carehome",
     # ["contact_participant_carehome", "contact_other_in_hh_carehome"])
+
+    df = assign_column_given_proportion(
+        df=df,
+        column_name_to_assign="ever_work_person_facing_or_social_care",
+        groupby_column="participant_id",
+        reference_columns=["work_social_care"],
+        count_if=[1, 2],
+    )  # not sure of correct  PIPELINE categories
```

Comment:
Not correct, these should use the strings associated with these values

Reply:
changed to pipeline names